nss-day-cohort-45/nutyshell-talking-to-strangers-1
# Rh news stuff

# Description
formatting of news cards
adding a new article is a form inside of a modal
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
- [] Bug fix (non-breaking change which fixes an issue)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
# Testing Instructions
fetch branch
serve
news should be formatted to be smaller cards
add a news article is now a modal
# Checklist:
- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] My changes generate no new warnings or errors
- [x ] I have added test instructions that prove my fix is effective or that my feature works
